src/plot.jl

```diff
@@ -148,40 +146,23 @@ function gplot(g::AbstractGraph{T},
     map!(z -> scaler(z, min_x, max_x), locs_x, locs_x)
     map!(z -> scaler(z, min_y, max_y), locs_y, locs_y)
 
-    # Determine sizes
-    #NODESIZE    = 0.25/sqrt(N)
-    #LINEWIDTH   = 3.0/sqrt(N)
-
-    max_nodesize = NODESIZE / maximum(nodesize)
-    nodesize *= max_nodesize
-    max_edgelinewidth = EDGELINEWIDTH / maximum(edgelinewidth)
-    edgelinewidth *= max_edgelinewidth
-    max_edgelabelsize = EDGELABELSIZE / maximum(edgelabelsize)
-    edgelabelsize *= max_edgelabelsize
-    max_nodelabelsize = NODELABELSIZE / maximum(nodelabelsize)
-    nodelabelsize *= max_nodelabelsize
+    # Scale sizes
+    nodesize *= (max_nodesize / maximum(nodesize))
+    edgelinewidth *= (max_edgelinewidth / maximum(edgelinewidth))
+    edgelabelsize *= (max_edgelabelsize / maximum(edgelabelsize))
+    nodelabelsize *= (max_nodelabelsize / maximum(nodelabelsize))
     max_nodestrokelw = maximum(nodestrokelw)
-    if max_nodestrokelw > 0.0
-        max_nodestrokelw = EDGELINEWIDTH / max_nodestrokelw
-        nodestrokelw *= max_nodestrokelw
+    if !iszero(max_nodestrokelw)
```

Comment:
Would testing for `max_nodestrokelw` not make more sense in case that `max_nodestrokelw` is zero?

Reply:
We are just catching for divide by 0 errors here I think. Passing negative values should work, they just don't get displayed. However, we can check for all inputs being positive when we do the other checks at the beginning of the function if that would be better.